Venmo/src/main/java/com/braintreepayments/api/VenmoResult.kt

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api
+
+/**
+ * Result of tokenizing a Venmo account
+ */
+sealed class VenmoResult {
```

Comment:
I dropped the `Payment` from this result object naming - open to opinions if it should be `VenmoPaymentResult` instead